src/expenses.json

```diff
@@ -0,0 +1 @@
+{"date":"asdas","title":"asdasd","amount":"asdads"}
```

Comment:
we need well formated JSON file, and question is - what if you sine form couple of times, new expenses will be added to file or it just overrrides every time?